docs/en/operations/settings/settings.md

```diff
@@ -979,6 +979,41 @@ Error count of each replica is capped at this value, preventing a single replica
 - [Table engine Distributed](../../operations/table_engines/distributed.md)
 - [`distributed_replica_error_half_life`](#settings-distributed_replica_error_half_life)
 
+
+## distributed_directory_monitor_sleep_time_ms {#distributed_directory_monitor_sleep_time_ms}
+
+Interval of data sending by the [Distributed](../table_engines/distributed.md) table engine. In case of any errors grows exponentially.
+
+Possible values:
+
+- Positive integer number of milliseconds.
+
+Default value: 100 milliseconds.
+
+
+## distributed_directory_monitor_max_sleep_time_ms {#distributed_directory_monitor_max_sleep_time_ms}
+
+Maximum interval of data sending by the [Distributed](../table_engines/distributed.md) table engine. Limits exponential growth of the interval set in the [distributed_directory_monitor_sleep_time_ms](#distributed_directory_monitor_sleep_time_ms) setting.
+
+Possible values:
+
+- Positive integer number of milliseconds.
+
+Default value: 30000 milliseconds.
+
+## distributed_directory_monitor_batch_inserts {#distributed_directory_monitor_batch_inserts}
+
+Enables/disables batch sending of inserted data.
+
+When batch sending is enabled, the [Distributed](../table_engines/distributed.md) engine tries to send multiple files of inserted data in one operation instead of sending them separately. Batch sending improves cluster performance by better local server and network resources utilization.
+
+Possible values:
+
+- 1 — Enabled.
+- 0 — Disabled.
+
+Defaule value: 0.
```

Comment:
Not clear why it's disabled by default if the documentation above shows only advantages.

Reply:
I don't know, really.
The current version of settings code https://github.com/ClickHouse/ClickHouse/blob/387cbca505c8afbe8d44a9b1da3377fdf3bbf5c5/dbms/src/Core/Settings.h#L85 set if to disabled by default.